### ISSUE TITLE
Not specifying a pac environment causes it to return an array not a hashtable

### ISSUE THREAD

**Issue by anwather** [Azure/enterprise-azure-policy-as-code#109]
If you run Build-DeploymentPlans and don't specify a PAC Environment - and you only have a single entry in the global-settings.json - the $pacEnvironment variable is returned as an array. This breaks Get-AzScopeTree and causes an error. 

If you specify a value for that parameter it returns a hash table correctly.

**Comment by techlake** [comment]
Yes, clearly a big (two-line, six-character) change :-(. I will add this to the v6.0 release tonight.

**Comment by anwather** [comment]
Thanks - the rest looks great - so much faster and nicer output too :)
